swagger.json

```diff
@@ -1121,6 +1121,122 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_certified_shifts/{time_ms}": {
+            "get": {
+                "tags": [
+                    "Fleet", "Drivers"
+                ],
+                "summary": "/fleet/drivers/{driver_id:[0-9]+}/hos_certified_shifts/{time_ms:[0-9]+}",
+                "description": "Fetch a certified shift for driver by driver_id and timestamp.",
+                "operationId": "get_hos_certified_shift",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                      "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                      "description": "ID (integer) of the driver with HOS logs.",
+                      "in": "path",
+                      "name": "driver_id",
+                      "required": true,
+                      "type": "integer"
+                    },
+                    {
+                        "description": "A specified timestamp within the bounds of a driver's certified shift, specified in milliseconds UNIX time.",
+                        "in": "path",
+                        "name": "time_ms",
+                        "required": true,
+                        "type": "integer"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns a certified shift for the driver, if one exists",
+                        "schema": {
+                            "$ref": "#/definitions/HosCertifiedShiftResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+              "summary": "/fleet/drivers/{driver_id:[0-9]+}/hos_certified_shifts/{time_ms:[0-9]+}",
```

Comment:
nit: There's a discrepancy between `get` request's `{time_ms}` vs the `patch` requests `{time_ms:[0-9]+}`. I think we should choose one and standardize.